src/com/android/calendar/CalendarToolbarHandler.java

```diff
@@ -141,7 +141,14 @@ private String buildDayOfWeek() {
             dayOfWeek = DateUtils.formatDateRange(mContext, mFormatter, mMilliTime, mMilliTime,
                     DateUtils.FORMAT_SHOW_WEEKDAY, mTimeZone).toString();
         }
-        return dayOfWeek.toUpperCase();
```

Comment:
I guess it's a language thing. Still, my proposal would be to only remove the `.toUpperCase()` as your solution is not portable among languages. Could you change your pull request accordingly?